q2_longitudinal/_utilities.py

```diff
@@ -400,40 +391,84 @@ def _regplot_subplots_from_dataframe(state_column, metric, metadata,
 
 
 def _control_chart_subplots(state_column, metric, metadata, group_column,
-                            ci=95, palette='Set1', plot_control_limits=True,
-                            xtick_interval=None):
+                            individual_id_column, ci=95, palette='Set1',
+                            plot_control_limits=True, xtick_interval=None,
+                            yscale='linear', spaghetti='no'):
 
     groups = metadata[group_column].unique()
-    states = metadata[state_column].unique()
-    chart, axes = plt.subplots(len(groups) + 1, figsize=(6, 18))
+    states = sorted(metadata[state_column].unique())
+    fig_count = len(groups) + 1
+    fig_height = fig_count * 6
```

Comment:
Maybe just inline this into the tuple, then the `6` is less suspicious in context.